scripts/openssl_env.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+OPENSSL_CMD=${OPENSSL_CMD:-openssl}
```

Comment:
@Baha-sk does this command need to be configurable?

Reply:
I generally think it’s a good pattern to allow overriding of commands in these scripts.

Reply:
I think it's a preemptive solution to a rare, user-specific problem, with the side effect of adding clutter to our scripts. Our build certainly doesn't require it - we have more control over this by using `frapsoft/openssl`.

That said, it's just a one-liner, although it leaves one (uselessly) wondering why it's configurable.